.devx-cicd/config.yaml

```diff
@@ -0,0 +1,12 @@
+build:
+  - image: 'adoptopenjdk/openjdk8:slim'
+    commands:
+      - './gradlew -x test'
```

Comment:
not directly related to this change
but using gradlew with docker images (or on demand jenkins slaves) means we always redownload the gradle distribution
we could probably try to cache gradle home  or part of it the same way it is done for .m2 (https://docs.gradle.org/current/userguide/gradle_wrapper.html)

Reply:
yes, it's an optimization we should be doing. Would you suggest to pre-build the docker image we use with that cache?